actionpack/lib/action_controller/caching.rb

```diff
@@ -46,6 +46,20 @@ def cache_configured?
         end
     end
 
+    # Cache or yield the block. The cache is suppose to never expire.
```

Comment:
This needs to communicate why you'd reach for this method and what it does. You have an HTTP response that never changes, so you want browsers and proxies to cache it indefinitely. On the first hit, we serve the response with long-lived cache freshness response headers. Then subsequent requests will always revalidate as fresh.

This method belongs elsewhere, too. This module is concerned with caching within the app (fragments, template dependencies, etc) not HTTP cache freshness.